scene/main/node.cpp

```diff
@@ -2092,7 +2092,7 @@ void Node::set_owner(Node *p_owner) {
 		_clean_up_owner();
 	}
 
-	ERR_FAIL_COND(p_owner == this);
+	ERR_FAIL_COND_MSG(p_owner == this, "Invalid owner. Owner cannot be self.");
```

Comment:
I'd suggest  `Attempted to set invalid owner (to [NodeName] if available). Owner cannot be self`. 
The difference is very slight, but I think using `attempt` indicates that although you tried, it didn't actually change the owner.

Reply:
This matches the error blow, so leaving it for consistency